Signal/translations/en.lproj/Localizable.strings

```diff
@@ -169,6 +169,9 @@
 /* Short name for edit menu item to save contents of media message. */
 "EDIT_ITEM_SAVE_ACTION" = "Save";
 
+/* Short name for edit menu item to share contents of media message. */
```

Comment:
nit: Do you want to re-use the existing "ATTACHMENT_VIEW_SHARE_ACTION" which has an identical translation and  description?